src/flamenco/runtime/program/fd_bpf_loader_serialization.c

```diff
@@ -224,6 +231,9 @@ fd_bpf_loader_input_deserialize_aligned( fd_exec_instr_ctx_t ctx,
     fd_borrowed_account_t * view_acc = NULL;
     /* Intentionally ignore the borrowed account error because it is not used */
     fd_instr_borrowed_account_view( &ctx, acc, &view_acc );
+    if( FD_UNLIKELY( !view_acc ) ) {
+      return FD_EXECUTOR_INSTR_ERR_INVALID_ACC_DATA;
+    }
```

Comment:
Doesn't this change the behaviour? Previously, if the account was unknown it was silently skipped, now we throw an error.

Reply:
We should not silently skip when we deserialize and can't find an account. This was intentional